swagger-gen.env

```diff
@@ -0,0 +1,5 @@
+# environment used for the openapi.json generation build step (see .csproj)
+
+ApplicationContext__UseInMemoryDb=true
+JWT__KEY=coolsecretkeywow
+JWT__ISSUER=leaderboards.gg
```

Comment:
(Can't get the "comment on this file" action to work)

Can't we just use `.env`? And side question; the `JWT_*` values here don't have to agree with what's in `.env`, right?

Reply:
`.env` is reserved for the developer's customized configuration, we can't use it.
This is a dummy configuration just to get the app to run without config validation errors basically.
Since Swagger generates from the actual API implementation, it needs to run Program.cs to setup the API to be able to detect the endpoints (I don't know the details).

Reply:
By the way if I had added the env variables to the `EnvironmentVariables` attribute [here](https://github.com/leaderboardsgg/leaderboard-backend/pull/139/files#diff-cd33d4cae6d4b44f2d39be061ba6f65da680c5b0c741b456743cdfb769a2719cR62), there would be no need for this file.

Reply:
Let's keep `swagger-gen.env`. It's easier to read the task.